policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit(Variable):
```

Comment:
make a az_property_tax_credit_eligible variable for the age and head eligibility